osgtest/tests/test_550_condorce.py

```diff
@@ -131,7 +138,13 @@ def test_07_ceview(self):
         try:
             src = core.to_str(urlopen(view_url).read())
         except EnvironmentError as err:
-            self.fail('Could not reach HTCondor-CE View at %s: %s' % (view_url, err))
+            debug_contents = 'Could not reach HTCondor-CE View at %s: %s\n%s\n' % (view_url, err, '=' * 20)
+            debug_file = '/var/log/condor-ce/CEViewLog'
+            try:
+                debug_contents += files.read(debug_file, True)
+            except FileNotFoundError:
+                debug_contents += 'Failed to read %s\n' % debug_file
+            self.fail(debug_contents)
```

Comment:
I'm not a fan of a log dump in a failure message. Failure messages should be limited to one line.

Just cat the file so it appears in its own section.

Reply:
> Just cat the file so it appears in its own section.

Like in a separate test or just in a separate command within the same test?

Reply:
I think a separate command is fine.

Reply:
Done